core/src/impl/Kokkos_GraphNodeImpl.hpp

```diff
@@ -144,6 +144,11 @@ struct GraphNodeImpl<ExecutionSpace, Kernel,
                 KernelDeduced&& arg_kernel)
       : base_t(ex), m_kernel((KernelDeduced&&)arg_kernel) {}
 
+  template <class KernelDeduced>
+  GraphNodeImpl(ExecutionSpace const& ex, _graph_node_then_ctor_tag,
+                KernelDeduced&& arg_kernel)
+      : base_t(ex), m_kernel((KernelDeduced&&)arg_kernel) {}
```

Comment:
```suggestion
      : base_t(ex), m_kernel(std::forward<KernelDeduced>(arg_kernel)) {}
```
(or keep it as is for consistency with the surrounding code?)

Reply:
I think they will prefer consistency.